salt/cloud/clouds/aliyun.py

```diff
@@ -339,7 +339,7 @@ def list_nodes_full(call=None):
         provider = comps[0]
 
     __opts__['update_cachedir'] = True
-    salt.utils.cloud.cache_node_list(ret, provider, __opts__)
+    __utils__['cloud.cache_node_list'](ret, provider, __opts__)
```

Comment:
If I'm not mistaken, since this is now being loaded by the loader we should have `__opts__` already packed into utils.If so, we should go ahead and modify these utility functions to simply use the copy of opts they already have, IMHO.